src/iotjs_binding.c

```diff
@@ -85,6 +86,42 @@ double iotjs_jval_as_number(jerry_value_t jval) {
 }
 
 
+bool iotjs_jval_get_data(jerry_value_t jval, iotjs_string_t* getbuffer) {
+  iotjs_bufferwrap_t* buffer_wrap = NULL;
+  if(jerry_value_is_string(jval)) {
```

Comment:
The spaces in the code are not consistent:
* Add space after the `if` keyword. (just like line 94).
* Add spaces after the comma.